libqtile/backend/wayland/qw/xdg-view.c

```diff
@@ -351,6 +391,7 @@ void qw_server_xdg_view_new(struct qw_server *server, struct wlr_xdg_toplevel *x
     xdg_view->xdg_toplevel = xdg_toplevel;
 
     xdg_view->base.shell = "XDG";
+    xdg_view->base.view_type = QW_VIEW_XDG;
```

Comment:
now we have shell and view_type, can we make them into one?

Reply:
Yes - but I'll tidy that up in a separate PR. I prefer the view type over the string.